README.md

```diff
@@ -2,12 +2,13 @@
 
 The workflow [msys2-htslib-build.yml][] builds a statically linked[^explanation] htslib.dll
 under msys2. See the [releases][] to download the tarball. To build a different
-version of htslib, update the env var `PACKAGE_VERSION`. To build an updated
+version of [htslib][], update the env var `PACKAGE_VERSION`. To build an updated
```

Comment:
```suggestion
version of [htslib], update the env var `PACKAGE_VERSION`. To build an updated
```

Reply:
Had to confirm I wasn't misremembering. The empty `[]` syntax is the canonical syntax, so it is guaranteed to work for all Markdown parsers.

https://daringfireball.net/projects/markdown/syntax#link

![image](https://github.com/user-attachments/assets/f0c6b597-b735-4f11-99d4-810f07d20741)

Though the syntax without the empty `[]` is included in the [CommonMark specification](https://spec.commonmark.org/0.31.2/) , refererred to as a [shortcut reference link](https://spec.commonmark.org/0.31.2/#shortcut-reference-link)

Reply:
TIL